spec/javascripts/test_spec.js

```diff
@@ -0,0 +1,14 @@
+/*global BlurredLocation */
+describe("Basic suite", function() {
+  "use strict";
+  var object = new BlurredLocation({
```

Comment:
Could we rename the instance `blurredLocation`? This is a convention and also makes code more readable.